src/CommunityToolkit/Xamarin.CommunityToolkit/Views/AvatarView/ImageSourceValidator.android.ios.macos.uwp.wpf.gtk.tizen.cs

```diff
@@ -31,16 +32,20 @@ namespace Xamarin.CommunityToolkit.UI.Views
 {
 	class ImageSourceValidator : IImageSourceValidator
 	{
-		public async Task<bool> IsImageSourceValidAsync(ImageSource source)
+#if MONOANDROID
+		Context context;
+#endif
+
+		public async Task<bool> IsImageSourceValidAsync(ImageSource source, Image image)
 		{
 			var handler = GetHandler(source);
 			if (handler == null)
 				return false;
-
 #if TIZEN
 			return await handler.LoadImageAsync(new NImage(XForms.NativeParent), source).ConfigureAwait(false);
 #elif MONOANDROID
-			var imageSource = await handler.LoadImageAsync(source, null).ConfigureAwait(false);
+			context ??= Platform.GetRenderer(image)?.View?.Context?.ApplicationContext;
```

Comment:
Do we need the application context here? On develop we have a helper method to grab the Context, can you add a todo here, to us replace this with the helper implementation?

Reply:
The good question whether we should use ApplicationContext or not. I don't know) But it works.
Also, we retrieve context only once now (??= operator). Do you think we should get Context every time this method invoked?

Reply:
No, because the ApplicationContext doesn't change... Also, on XF applications the ViewContext is the same, in most cases. Let move forward with this and if someone complains in the future we change it.

Reply:
@pictos sounds good